internal/distro/fedora31/distro.go

```diff
@@ -23,7 +23,8 @@ type Fedora31 struct {
 	buildPackages []string
 }
 
-type imageType struct {
```

Comment:
You could actually keep the name `imageType` (and same for `arch`, i just realised now), that would make it easier to copy into subsequent distros.